x/concentrated-liquidity/swaps_test.go

```diff
@@ -754,9 +869,14 @@ func (s *KeeperTestSuite) TestCalcAndSwapInAmtGivenOut() {
 			priceLimit:       sdk.NewDec(4094),
 			expectedTokenOut: sdk.NewCoin("eth", sdk.NewInt(2000000)),
 			expectedTokenIn:  sdk.NewCoin("usdc", sdk.NewInt(9103425685)),
-			expectedTick:     sdk.NewInt(300952),
-			newLowerPrice:    sdk.NewDec(4000),
-			newUpperPrice:    sdk.NewDec(4545),
+			// crosses one tick during swap that has fee growth outside. Thus 3-1 = 2*DefaultFeeAccumCoins expected in fee accumulator
+			expectedTick:                     sdk.NewInt(300952),
+			expectedLowerTickFeeGrowth:       DefaultFeeAccumCoins.MulDec(sdk.NewDec(2)),
+			expectedUpperTickFeeGrowth:       DefaultFeeAccumCoins.MulDec(sdk.NewDec(2)),
```

Comment:
This comment makes more sense than the previous ones but I am not seeing where 3-1 comes from.

Reply:
ditto for above ^! Changed as well! Let me know if this makes more sense now!